library-files.s6l

```diff
@@ -1,27 +1,8 @@
 #!r6rs
-;; Copyright (c) 2010 Derick Eddington
-;;
-;; Permission is hereby granted, free of charge, to any person obtaining a
-;; copy of this software and associated documentation files (the "Software"),
-;; to deal in the Software without restriction, including without limitation
-;; the rights to use, copy, modify, merge, publish, distribute, sublicense,
-;; and/or sell copies of the Software, and to permit persons to whom the
-;; Software is furnished to do so, subject to the following conditions:
-;;
-;; The above copyright notice and this permission notice shall be included in
-;; all copies or substantial portions of the Software.
-;;
-;; Except as contained in this notice, the name(s) of the above copyright
-;; holders shall not be used in advertising or otherwise to promote the sale,
-;; use or other dealings in this Software without prior written authorization.
-;;
-;; THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-;; IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-;; FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL
-;; THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-;; LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
-;; FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
-;; DEALINGS IN THE SOFTWARE.
+
+;;;; SPDX-FileCopyrightText: 2010 Derick Eddington
+;;;;
+;;;; SPDX-License-Identifier: MIT
```

Comment:
This changes the license by removing the "Except as contained in this notice..." paragraph.  Since this SRFI was published after 2005, it wasn't part of the Great Relicensing, so we can't assume that the author agreed to use the plain MIT license.

Reply:
I've reached out to them (your are CC'd).  Let's see.

Reply:
They've answered favorably to use the plain MIT instead of the variant of MIT they had used.  I've mentioned in the commit the change is made with their permission.